store/copr/mpp.go

```diff
@@ -79,7 +70,7 @@ func (c *MPPClient) ConstructMPPTasks(ctx context.Context, req *kv.MPPBuildTasks
 		tasks, err = buildBatchCopTasksForPartitionedTable(ctx, bo, c.store, rangesForEachPartition, kv.TiFlash, true, ttl, true, 20, partitionIDs)
 	} else {
 		if req.KeyRanges == nil {
-			return c.selectAllTiFlashStore(), nil
+			return nil, errors.New("KeyRanges in MPPBuildTasksRequest is nil")
```

Comment:
Seems it's a different behavior, a request without key ranges is an invalid usage, so returns error is ok, right?

Reply:
Yes